Workbooks/Virtual Machines - Performance Analysis/Performance Analysis for a Group of VMs (Dev)/Performance Analysis for a Group of VMs (Dev).workbook

```diff
@@ -8,286 +8,240 @@
         "crossComponentResources": [
           "value::selected"
         ],
+        "parameters": [],
+        "style": "above",
+        "queryType": 1,
+        "resourceType": "microsoft.resourcegraph/resources"
+      },
+      "conditionalVisibility": {
+        "parameterName": "_",
+        "comparison": "isEqualTo",
+        "value": "_"
+      },
+      "name": "parameters - 0"
```

Comment:
in general you guys should be setting names on all the steps, especially query/params steps because those can be found in telemetry for your queries, to identify which queries are slow/failing